charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -244,16 +244,24 @@ scrape_configs:
 - job_name: 'blackbox-ingress'
   metrics_path: /probe
   params:
-    module: [http_2xx]  # Look for a HTTP 200 response.
+    # Look for a HTTP 200 response per default.
+    # Can be overwritten by annotating the ingress resource with the expected return codes, e.g. `prometheus.io/probe_code: "4xx"`
```

Comment:
module annotation on ingress e.g. `prometheus.io/probe_code: "4xx"`. `2xx` is default.